exercises/roman-numerals/src/example.c

```diff
@@ -0,0 +1,43 @@
+#include "roman_numerals.h"
+#include <stdlib.h>
+#include <string.h>
+
+#define NUM_OF_ELEMENTS(a)    (sizeof(a) / sizeof(a[0]))
+#define MAX_NUMERAL_LENGTH    (6)
+
+typedef struct
+{
+    char *numeral;
+    unsigned int value;
+} Numeral_Values_t;
```

Comment:
Serious nitpick: how about `numeral_values_t` or `numeralvalues_t` to match the rest of the style conventions?